database/connector.py

```diff
@@ -4,6 +4,7 @@
 import inspect
 import os
 import time
+from dataclasses import asdict
```

Comment:
not supported for python3.5 and python3.6 ! started since python3.7 onwards(https://docs.python.org/3.7/library/dataclasses.html)

@ChakshuGupta : sorry for late reply. is there any other change which can be done which could support python3.5 and higher versions too?
if not 
@Ali-Razmjoo : do you think we should only support this project python3.7 onwards to support this feature?

Reply:
I found a solution. I'll push that in a bit!